package.json

```diff
@@ -14,63 +14,72 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint:* test:*",
+    "test": "npm-run-all lint test:*",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.19.0",
-    "ember-cli-htmlbars": "^4.3.1"
+    "ember-cli-babel": "^7.23.0",
+    "ember-cli-typescript": "^4.1.0"
```

Comment:
should this be in `devDependencies`?

Reply:
@maxfierke good point. As this addon has only js files in `/addon` folder, I think it actually should be in `devDependencies `.

Got confused by `ember-cli-typescript` complaint:
```
Run yarn ember build --environment test
yarn run v1.22.4
WARNING: `ember-cli-typescript` should be included in your `dependencies`, not `devDependencies`
```

will revert

Reply:
cc @jamescdavis can you confirm?

Reply:
Yeah, that's technically correct. There are no .ts files that consumers need to transpile so it can stay in devDeps. Maybe we can add a flag to suppress that warning for cases like this where it really does only apply to test files.